cdec/update-server-packaging-war/src/main/java/com/codenvy/im/update/ReportService.java

```diff
@@ -47,18 +51,29 @@ public ReportService() {
     @GET
     @Path("/parameters/{report_type}")
```

Comment:
What kind of authorisation is used for this class? Who can call methods of this class?